35 send registration emails

hubertborysowski opened the pull request:
# Registration Email Verification
This service provides an email verification by sending verification emails in registration process.
## Workflow Checklist
- [x] 🫥 Insert Dependencies 
- [x] ✉️ Create Working Basic Email Service 
- [ ] 🔬 Create Tests 
- [x] 🌐 Create URL Generator
- [x] 📨 Send Verification Email 
- [ ] 📄 Create Documentation 
- [ ] ✅ Ready for review!

hubertborysowski committed:
[38] Spring Mail Dependencies and Basic Email Service ✉️
Basic email service is still not working;

hubertborysowski committed:
[38] Basic Email Service ✉️
Emails are now sent.

hubertborysowski commented:
My bad! By mistake I named 2 of my commits using [38] number which belongs to different branch. Next commits are going to be named correctly❗

hubertborysowski committed:
[35] .gitignore update ⬆️

JSON dependency has been removed as is no longer in use.

Mateusz-Dobrzynski commented:
I like the checklist that you included in the PR. We might consider including something similar in a PR template. I suggest, however, moving the test creation up in the list, as this encourages Tests Driven Development.

hubertborysowski committed:
[35] .gitignore update ⬆️

.config dependency has been added.

hubertborysowski committed:
[35] Merge [38] to [35] 🌌

hubertborysowski committed:
[35] Email Sent After Successful Registration 📨

hubertborysowski committed:
[35] Poperties update ⬆️

hubertborysowski committed:
Merge branch 'dev' into 35-send-registration-emails

hubertborysowski committed:
[35] Sending E-mail With Verification Link 📬
The Service is still not automated as automatic link generation with
token, userId, domain name / IP address and port are still added by hand.

hubertborysowski committed:
[35]  .gitignore update ⬆️

hubertborysowski committed:
[35] Verification URL Generation (Part1) 🔍

hubertborysowski committed:
[35] Update .gitignore ⬆️

hubertborysowski committed:
[35] Getting Email Data From .config 🫳
